plugins/@grouparoo/hubspot/src/lib/export/destinationMappingOptions.ts

```diff
@@ -1,12 +1,18 @@
-import { DestinationMappingOptionsMethod } from "@grouparoo/core";
+import {
+  DestinationMappingOptionsMethod,
+  DestinationMappingOptionsResponseTypes,
+  SimpleAppOptions,
+} from "@grouparoo/core";
 import { connect } from "./../connect";
 
+const importantFields = ["start_date", "firstname", "lastname", "company"];
```

Comment:
`start_date` looks to be specific to some sort of Integration, so probably not important.